index.js

```diff
@@ -1,10 +1,11 @@
 
+
 const axios = require('axios');
 const ua = require('universal-analytics');
 const Discord = require('discord.js');
 const Snoowrap = require('snoowrap');
 const { CommentStream } = require('snoostorm');
-
+const {aliasingLightning} = require('./utils/aliasingLightning')
```

Comment:
fix spacing, and name, and function location -

- should be in shared.js
- call it `aliasLightning` (this tense makes more sense)
- follow spacing of other imports